proto/rollkit/rollkit.proto

```diff
@@ -49,24 +49,29 @@ message Header {
 	// pubkey can't be recovered by the signature (e.g. ed25519).
 	bytes proposer_address = 10;
 
-	// public key of the proposer so we can verify the signature
-	bytes proposer_public_key = 11;
-
-
 	// Hash of block aggregator set, at a time of block creation
-	bytes aggregators_hash = 12;
+	bytes aggregators_hash = 11;
 
 	// Chain ID the block belongs to
-	string chain_id = 13;
+	string chain_id = 12;
 }
 
 message Commit {
 	repeated bytes signatures = 1;
 }
 
+message Validator {
+	bytes public_key = 1;
+}
+
+message ValidatorSet {
+	repeated Validator validators = 1;
+}
+
```

Comment:
> [non-blocking][question] It still feels that we might need to put the sequencer set in the header as this is something the proposer can set up arbitrarily, making the signatures useless before the fraud-proof window is closed, as setting a wrong validator set could be fraud itself. This is fine for now but will need discussion as it might be security relevant. I would propose keeping #784 open until we resolve this question.

if the sequencer alters the signer keys before gossip, it can be considered fraud, which is not different from the sequencer altering the signature. both will be caught by the light node which fails to verify the signatures? i am not seeing the point of what is special with signing key altering fraud?

Reply:
> Maybe also change the name to Aggreagtor as we have too many name inconsistencies. (aggregator , Block proposer, validator, sequencer ) . We should open a follow-up issue resolving these name differences.

we should take this up in a separate issue.

Reply:
https://github.com/rollkit/rollkit/issues/796